test/e2e/tests/state-snapshots/errors-after-init-opt-in-ui-state.json

```diff
@@ -116,8 +116,8 @@
     "currencyRates": {
       "ETH": {
         "conversionDate": "number",
-        "conversionRate": 1700,
-        "usdConversionRate": 1700
+        "conversionRate": 1300,
+        "usdConversionRate": 1300
```

Comment:
Fixes failing e2e which undoes the value change in the offending PR. This test changing should have raised a flag for me to investigate further.

https://github.com/MetaMask/metamask-extension/pull/21549/files#diff-16a567d598793bb9001d6c8c282cfd1b06f9ed629620a63355bd7c24d78e637dR116